MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,429 @@
+//
+//  MovieDetailsViewController.swift
+//  MovieApp
+//
+//  Created by Bruno Busic on 30.03.2022..
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class MovieDetailsViewController: UIViewController {
+    
+    
+    /*
+     Problemi:
+     kad rotiram screen sve mi se unisti
+     u posteru trebam labele dodavati odozdola prema gore
+     kako da dodam ou slicicu favorite
+     kad stavim label.top to bottom of another label ne rade mi inseti i ofseti
+     */
+    
+    
+    
+    private var safeAreaView: UIView!
+    private var posterView: UIView!
+    private var scrollView: UIScrollView!
+    private var contentView: UIView!
+    
+    private var posterImage: UIImageView!
+    private var popularityLabel: UILabel!
+    private var userScoreLabel: UILabel!
+    private var titleLabel: UILabel!
+    private var releaseYearLabel: UILabel!
+    private var releaseCountryLabel: UILabel!
+    private var releaseDateLabel: UILabel!
+    private var genreLabel: UILabel!
+    private var durationLabel: UILabel!
+    private var favoriteView: UIView!
+    private var favoriteImage: UIImageView!
+    
+    private var overviewLabel: UILabel!
+    private var overviewTextLabel: UILabel!
+    private var caracter1: UILabel!
```

Comment:
What if I tell you to add 5 more people? That is 10 more labels and you have to constrain all of them. A better solution would be for you to extract this section into a separate class that extends UIView and you could reuse it. Also, you have to constrain the next view to the previous. There is a component that does that work for you: UIStackView. Try to implement the described solution.